embedded-graphics/src/style/primitive_style.rs

```diff
@@ -79,6 +79,63 @@ where
 }
 
 /// Primitive style builder.
+///
+/// Use this builder to create styles for [primitives]. For convenience, the [`primitive_style!`]
+/// macro is also provided.
+///
+/// If any properties on the builder are ommitted, the value will default to `None` (transparent).
```

Comment:
```suggestion
/// If any properties on the builder are omitted, the value will default to `None` (transparent).
```

Not all values default to `None`, like `stroke_width`.